docs/src/TerminalVelocityComparisons.jl

```diff
@@ -78,95 +89,134 @@ function rain_terminal_velocity_individual_SB(
-    label = "Rain-SB2006 [ND]",
-)
-PL.plot!(
-    q_rai_range * 1e3,
-    [
-        CM2.rain_terminal_velocity(param_set, Chen2022, q_rai, ρ_air, N_rai)[1]
-        for q_rai in q_rai_range
-    ],
-    linewidth = 3,
-    xlabel = "q_rain [g/kg]",
-    ylabel = "terminal velocity [m/s]",
-    label = "Rain-Chen2022 [ND]",
-)
-PL.plot!(
-    q_rai_range * 1e3,
-    [
-        CM2.rain_terminal_velocity(param_set, SB2006, q_rai, ρ_air, N_rai)[2]
-        for q_rai in q_rai_range
-    ],
-    linewidth = 3,
-    xlabel = "q_rain [g/kg]",
-    ylabel = "terminal velocity [m/s]",
-    label = "Rain-SB2006 [M]",
-)
-PL.plot!(
-    q_rai_range * 1e3,
-    [
-        CM2.rain_terminal_velocity(param_set, Chen2022, q_rai, ρ_air, N_rai)[2]
-        for q_rai in q_rai_range
-    ],
-    linewidth = 3,
-    xlabel = "q_rain [g/kg]",
-    ylabel = "terminal velocity [m/s]",
-    label = "Rain-Chen2022 [M]",
-)
-PL.plot!(
-    q_rai_range * 1e3,
-    [
-        CM1.terminal_velocity(param_set, rain, ρ_air, q_rai) for
-        q_rai in q_rai_range
-    ],
-    linewidth = 3,
-    xlabel = "q_rain [g/kg]",
-    ylabel = "terminal velocity [m/s]",
-    label = "Rain-Ogura-1-moment",
-)
-
-PL.title!("Terminal Velocity vs. Specific Humidity of Rain", titlefontsize = 9)
-PL.savefig("terminal_velocity_bulk_comparisons.svg")
-
-PL.plot(
-    D_r_range,
-    [
-        rain_terminal_velocity_individual_SB(param_set, ρ_air, D_r) for
-        D_r in D_r_range
-    ],
-    linewidth = 3,
-    xlabel = "D [m]",
-    ylabel = "terminal velocity [m/s]",
-    label = "Rain-SB2006",
-)
-PL.plot!(
-    D_r_range,
-    [
-        rain_terminal_velocity_individual_Chen(param_set, ρ_air, Chen2022, D_r)
-        for D_r in D_r_range
-    ],
-    linewidth = 3,
-    xlabel = "D [m]",
-    ylabel = "terminal velocity [m/s]",
-    label = "Rain-Chen2022",
-)
-PL.title!("Terminal Velocity vs. Diameter", titlefontsize = 13)
-PL.savefig("terminal_velocity_individual_raindrop.svg")
+"""
+    terminal_velocity_individual_1M(prs, precip, ρ, D_r)
+
+ - `prs` - set with free parameters
+ - `precip` - precipitation type (rain or snow)
+ - `ρ` - air density
+ - `D_r` - particle diameter
+
+Returns the fall velocity of a raindrop or snow from 1-moment scheme
+"""
+function terminal_velocity_individual_1M(
+    prs::APS,
+    precip::CMT.AbstractPrecipType,
+    ρ::FT,
+    D_r::FT,
+) where {FT <: Real}
+    _χv::FT = CM1.χv(prs, precip)
+    _v0::FT = CM1.v0(prs, ρ, precip)
+    _ve::FT = CM1.ve(prs, precip)
+    _r0::FT = CM1.r0(prs, precip)
+    _Δv::FT = CM1.Δv(prs, precip)
+    vt = _χv * _v0 * (D_r / (2 * _r0))^(_Δv + _ve)
+    return vt
+end
+
+#TODO - add ice_terminal_velocity_individual
+
+"""
+    snow_terminal_velocity_individual_Chen(prs, ρ, D_r)
+
+ - `prs` - set with free parameters
+ - `ρ` - air density
+ - `D_r` - particle diameter
+
+Returns the fall velocity of snow from Chen et al 2022
+"""
+function snow_terminal_velocity_individual_Chen(
```

Comment:
It would be nice to have the function throw a warning or error if D_r >= 0.625mm. If you do end up doing that, make sure to add a gpu/unit test for it

Reply:
This is just for plotting. We don't use it in the actual code. - In the code we use the mass weighted mean. And the integral is done from zero to infinity, so I guess we are using the function outside of it's range. I think it's fine tho